app/forms/SubmissionTypeFormProvider.scala

```diff
@@ -19,27 +19,38 @@ package forms
 import play.api.data.Form
 import play.api.data.Forms._
 import play.api.data.format.Formatter
-import utils.RadioOption
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.viewmodels.radios.RadioItem
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.Text
+import uk.gov.hmrc.govukfrontend.views.viewmodels.label.Label
 
 object SubmissionTypeFormProvider extends FormErrorHelper {
 
-  def submissionTypeFormatter = new Formatter[String] {
+  def submissionTypeFormatter(implicit messages: Messages) = new Formatter[String] {
     def bind(key: String, data: Map[String, String]) = data.get(key) match {
       case Some(s) if optionIsValid(s) => Right(s)
-      case None => produceError(key, "error.required")
+      case None => produceError(key, "error.select.option")
       case _ => produceError(key, "error.unknown")
     }
 
     def unbind(key: String, value: String) = Map(key -> value)
   }
 
-  def apply(): Form[String] =
-    Form(single("value" -> of(submissionTypeFormatter)))
-
-  def options = Seq(
-    RadioOption("submissionCategory", "council_tax_webform"),
-    RadioOption("submissionCategory", "council_tax_xml_upload")
+  def apply()(implicit messages: Messages): Form[String] =
+    Form(single("submissionCategory" -> of(submissionTypeFormatter)))
+
+  def options(implicit messages: Messages)  = Seq(
+    RadioItem(
+      content = Text(messages("submissionCategory.council_tax_webform")),
```

Comment:
What is difference between `label` and `content`. I'm using only `content`

Reply:
I use the content to set label of each radio button but in reading for accessibility there should be a label tag within the each radio button which is used by the screen readers. This is linked back to the radio button using the id. https://www.w3schools.com/tags/tag_label.asp